### PR TITLE
readbility

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

File: src/lxqtplatformtheme.cpp [hunk 1]
```diff
@@ -149,7 +147,7 @@ void LXQtPlatformTheme::loadSettings() {
     color = highlightedTextColor_;
     highlightedTextColor_.setNamedColor(settings.value(QLatin1String("highlighted_text_color")).toString());
     if (!paletteChanged_)
-        paletteChanged_ = color != highlightedTextColor_.isValid() && color != highlightedTextColor_;
+        paletteChanged_ = color != static_cast<QRgb>(highlightedTextColor_.isValid()) && color != highlightedTextColor_;
```
Comment (tsujan) [inline src/lxqtplatformtheme.cpp:150]: This was my typo, I think. It should be:

```c++
paletteChanged_ = highlightedTextColor_.isValid() && color != highlightedTextColor_;
```

Comment (luis-pereira) [reply src/lxqtplatformtheme.cpp:150]: Thanks. Fixed.

Comment (tsujan) [reply src/lxqtplatformtheme.cpp:150]: We wouldn't find this mistake of mine if it weren't for your patch.

BTW, you forgot to change it.

Comment (luis-pereira) [reply src/lxqtplatformtheme.cpp]: @tsujan Done in https://github.com/lxqt/lxqt-qtplugin/pull/66.

**Review by tsujan — APPROVED** [review]
GTM

**Pull request merged**